tox.ini

```diff
@@ -30,6 +30,5 @@ change_dir = source/infrastructure
 description = run CDK unit tests
 package = skip
 commands =
-    npm install
-    npm run build
+    npm ci
```

Comment:
I'm not totally sure you want `ci` in here.`ci` is a clean install, which is good for github workflows and pipelines, but it's actually slower than `install` because of that. In ASR, `npm install` after already installing takes less than a second, while `ci` takes ~20 seconds. You're looking at a longer local test cycle. I'm not sure what the right way to decouple the local workflow from the github workflow is. Maybe ask Nathaniel.

Reply:
yeah fair enough, I'll leave it as npm install and cut the run build.

difference between using ci and install appears to be ~30-40 and ~20 seconds respectively. I wonder if there's anything else we can do to bring that time down...